pkg/csi/cinder/nodeserver_test.go

```diff
@@ -335,10 +301,19 @@ func TestNodeExpandVolume(t *testing.T) {
 	// Init assert
 	assert := assert.New(t)
 
+	// setup for test
+	tempDir := os.TempDir()
+	defer os.Remove(tempDir)
+	volumePath := filepath.Join(tempDir, FakeTargetPath)
+	err := os.MkdirAll(volumePath, 0750)
+	if err != nil {
+		t.Fatalf("Failed to set up volumepath: %v", err)
+	}
```

Comment:
```suggestion
	// setup for test
	tempDir := os.TempDir()
	volumePath := filepath.Join(tempDir, FakeTargetPath)
	err := os.MkdirAll(volumePath, 0750)
	if err != nil {
		t.Fatalf("Failed to set up volumepath: %v", err)
	}
	defer os.RemoveAll(tempDir)
```

Reply:
defer os.RemoveAll(tempDir) good catch. defer os.RemoveAll(tempDir) -> defer os.RemoveAll(volumePath)

os.TempDir() - returns `/tmp` folder, it does not create any folders, os.Remove(tempDir) can remove `/tmp` during the tests...